From 95c99ae24e8772de09a24925b8fd28da89f40693 Mon Sep 17 00:00:00 2001
From: Samuel Mehrbrodt <Samuel.Mehrbrodt@cib.de>
Date: Mon, 11 May 2020 07:48:23 +0200
Subject: Fix 32bit linux build

Change-Id: I4faf3fb20c632163f98264d162bbf85f80b3603d
Reviewed-on: https://gerrit.libreoffice.org/c/core/+/93797
Tested-by: Jenkins
Reviewed-by: Samuel Mehrbrodt <Samuel.Mehrbrodt@cib.de>
(cherry picked from commit fa0d576378cafab396a3fb75b1dbe1905d5df9ce)
Reviewed-on: https://gerrit.libreoffice.org/c/core/+/96090
Reviewed-by: Michael Stahl <michael.stahl@cib.de>
(cherry picked from commit c7bb5be1b6a9d6ea8a3fb10dba32c2fd346261a3)
Reviewed-on: https://gerrit.libreoffice.org/c/core/+/96095
Reviewed-by: Stephan Bergmann <sbergman@redhat.com>
Tested-by: Stephan Bergmann <sbergman@redhat.com>
---
 toolkit/qa/cppunit/EventContainer.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/toolkit/qa/cppunit/EventContainer.cxx b/toolkit/qa/cppunit/EventContainer.cxx
index 300c8e5adb74..97a125c60824 100644
--- a/toolkit/qa/cppunit/EventContainer.cxx
+++ b/toolkit/qa/cppunit/EventContainer.cxx
@@ -70,7 +70,7 @@ CPPUNIT_TEST_FIXTURE(EventContainerTest, testInsertOrder)
 
     Sequence<OUString> aEventNames(xEvents->getElementNames());
     sal_Int32 nEventCount = aEventNames.getLength();
-    CPPUNIT_ASSERT_EQUAL(4, nEventCount);
+    CPPUNIT_ASSERT_EQUAL(static_cast<sal_Int32>(4), nEventCount);
 
     CPPUNIT_ASSERT_EQUAL(OUString("b"), aEventNames[0]);
     CPPUNIT_ASSERT_EQUAL(OUString("a"), aEventNames[1]);
-- 
cgit v1.2.1

